test/controllers/stacks_controller_test.rb

```diff
@@ -166,5 +166,20 @@ class StacksControllerTest < ActionController::TestCase
       get :lookup, id: @stack.id
       assert_redirected_to stack_path(@stack)
     end
+
+    test "#create should not create stack with invalid deploy_url" do
```

Comment:
please don't use `should` in test names. be imperative about what it does. `#create does not create ...`